packages/config-array/tests/config-array.test.js

```diff
@@ -2334,14 +2922,11 @@ describe("ConfigArray", () => {
 					);
 				});
 
-				it("should return false when first-level subdirectories are ignored with leading slash and then one is negated", () => {
```

Comment:
This test was bogus because the first ignore pattern was not ignoring the subdirectory path, so I removed the negated pattern and changed the description to match what is going on.